cleanup_pr.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+#
+# GitHub App for the EESSI project
+#
+# A bot to help with requests to add software installations to the EESSI software layer,
+# see https://github.com/EESSI/software-layer
+
+# This script cleans up (deletes) all build artefacts and temporary storage tarballs of a given PR
+#
+# author: Thomas Roeblitz (@trz42)
+#
+# license: GPLv2
+#
+
+SCRIPT_DIR=$(dirname $(realpath $BASH_SOURCE))
+
+function display_help
+{
+  echo "Usage: $0 [OPTIONS] <PR number>"                                                >&2
+  echo "  -b | --jobs-base-dir DIRECTORY  -  jobs base directory [default: reads"       >&2
+  echo "                                     value from bot config file app.cfg or .]"  >&2
+  echo "  -D | --dry-run                  -  only show commands that would be run"      >&2
+  echo "                                     [default: false]"                          >&2
+  echo "  -h | --help                     -  display this usage information"            >&2
+}
+
+function get_jobs_base_dir
+{
+  app_cfg_path=${1}
+  grep jobs_base_dir ${app_cfg_path} | grep -v '^[ ]*#' | sed -e 's/^[^=]*=[ ]*//'
+}
+
+echo
+
+if [[ $# -lt 1 ]]; then
+    display_help
+    exit 1
+fi
+
+# process command line args
+POSITIONAL_ARGS=()
+
+jobs_base_dir=
+dry_run=false
+
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    -b|--jobs-base-dir)
+      if [[ $# -gt 1 ]]; then
+        jobs_base_dir="$2"
+        shift 2
+      else
+        echo "Error: missing argument (directory) for parameter '${1}'"
+        exit 2
+      fi
+      ;;
+    -D|--dry-run)
+      dry_run=true
+      shift 1
+      ;;
+    -h|--help)
+      display_help
+      exit 0
+      ;;
+    -*|--*)
+      echo "Error: Unknown option: $1" >&2
+      exit 1
+      ;;
+    *)  # No more options
+      POSITIONAL_ARGS+=("$1") # save positional arg
+      shift
+      ;;
+  esac
+done
+
+# restore potentially parsed filename(s) into $*
+set -- "${POSITIONAL_ARGS[@]}"
+
+if [[ $# -lt 1 ]]; then
```

Comment:
Don't we want exactly 1 PR number here?

If so, use `$# -ne 1`, and adjust the error message, like `"ERROR: Exactly one PR number should be provided"`.

Reply:
Done in https://github.com/EESSI/eessi-bot-software-layer/pull/217/commits/1f9c0dab85e3c4cceb984a968753533ceeed7f84